Original patch from: gentoo/src/patchsets/glibc/2.9/5063_all_glibc-dont-build-timezone.patch

-= BEGIN original header =-
timezone data has been split into the package sys-libs/timezone-data

-= END original header =-

diff -durN glibc-2_9.orig/Makeconfig glibc-2_9/Makeconfig
--- glibc-2_9.orig/Makeconfig	2009-02-02 22:01:05.000000000 +0100
+++ glibc-2_9/Makeconfig	2009-02-02 22:01:27.000000000 +0100
@@ -919,7 +919,7 @@
 	      stdlib stdio-common libio malloc string wcsmbs time dirent    \
 	      grp pwd posix io termios resource misc socket sysvipc gmon    \
 	      gnulib iconv iconvdata wctype manual shadow po argp	    \
-	      crypt nss localedata timezone rt conform debug		    \
+	      crypt nss localedata rt conform debug		    \
 	      $(add-on-subdirs) $(dlfcn) $(binfmt-subdir)
 
 ifndef avoid-generated
